slstatus.c

```diff
@@ -811,7 +811,8 @@ main(int argc, char *argv[])
 				element = smprintf(UNKNOWN_STR);
 				warnx("Failed to format output");
 			}
-			strlcat(status_string, element, sizeof(status_string));
+			if (strlcat(status_string, element, sizeof(status_string)) >= sizeof(status_string))
+				warnx("Output too long");
```

Comment:
My coding style is to use braces also for single-line statements. Could you please add these?